common/protos/proto/config.proto

```diff
@@ -18,6 +18,7 @@ package databend_proto;
 
 message S3StorageConfig {
   uint64 version = 100;
+  uint64 min_compatible = 101;
```

Comment:
Why `min_compatible` is 101 bigger then `version`?

Reply:
This is the index of a field. not the value of it

Reply:
😂, My fault.